core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml

```diff
@@ -34,9 +34,11 @@
     dest: /tmp/vault_helm_chart_values.yaml
   when: vault_helm_chart_values_bool
 
-- name: Install and configure Hashicorp Vault if enabled
+- name: Install and configure Hashicorp Vault if enabled and no cloud service is used
   when:
     - specification.vault_enabled
+    - k8s_as_cloud_service is not defined
```

Comment:
I think this line has no effect, we can safely assume this is defined always (it's merged with defaults), no? :thinking: 
```suggestion
```

Reply:
Also it could mask some future issues with upgrades... :thinking: